src/System.Memory/tests/Performance/Perf.ReadOnlySequence.GetPosition.cs

```diff
@@ -0,0 +1,224 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Memory.Tests;
+using Microsoft.Xunit.Performance;
+using Xunit;
+
+namespace System.Buffers.Tests
+{
+    public class Rerf_ReadOnlySequence_GetPosition
+    {
+        private const int InnerCount = 10_000;
+        volatile static int _volatileInt = 0;
+
+        [Benchmark(InnerIterationCount = InnerCount)]
+        [InlineData(10_000, 100)]
+        private static void Byte_Array(int bufSize, int bufOffset)
+        {
+            var buffer = new ReadOnlySequence<byte>(new byte[bufSize], bufOffset, bufSize - 2 * bufOffset);
+            int offset = (int)buffer.Length / 10;
+
+            foreach (BenchmarkIteration iteration in Benchmark.Iterations)
+            {
+                int localInt = 0;
+                using (iteration.StartMeasurement())
+                {
+                    for (int i = 0; i < Benchmark.InnerIterationCount; i++)
+                    {
+                        SequencePosition p = buffer.Start;
+                        for (int j = 0; j < 10; j++)
```

Comment:
Do not use inner loops like this. Increase the inner iteration count if necessary.

Reply:
This loop to count position to buffer.End I can not replace it by `while (pos != buffer.End)` because we return flags it Start and End and equality comparison does not works. I suggested to remove flags from Start and End but it is in discussion and not implemented yet.

Reply:
Ah I see. Then, its fine to leave it as is.